lib/compilers/hlsl.js

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2021, Compiler Explorer Authors
```

Comment:
Header date is wrong

Reply:
Thanks this was taken from a different file. Random question: Are these copyright headings needed for this project in general?